### DISCUSSION TITLE
Unable to run 'install' successfully

### DISCUSSION THREAD

**Discussion by joelbudreau** [derekantrican/GAS-ICS-Sync#324] [Q&A]
### The problem

I'm unable to run 'install' successfully.

1. I create a copy of https://script.google.com/home/projects/1BOk8MDLbLaHh6SwG1M1tsgNXjkcC-79LE0QoipRuTDxbO3fMVvqoROQD while logged in under my chome/gmail account.
2. I change the 3 template calendar lines so that only my 1 calendar is listed `[<my .ics>, "my calendar"]`
3. Run (install)

<img width="1052" alt="Screenshot 2023-03-17 at 10 13 30 AM" src="https://user-images.githubusercontent.com/1182950/225973265-0d08448a-0e01-45c3-917d-1ebcc0426e9b.png">

I try to Debug the install() function, but I get an error there as well:
<img width="1107" alt="Screenshot 2023-03-17 at 10 15 28 AM" src="https://user-images.githubusercontent.com/1182950/225973751-c3d82784-fad9-4c6b-8ecf-049a41eea159.png">

So:
- What am I missing?
- Am I supposed to be seeing some kind of authentication request pop up? (I'm not seeing anything like that)

Thanks for the help!

### Version of GAS-ICS-Sync

5.7

**Comment by derekantrican** [comment]
Can you try creating a new copy? That "creating the Cloud Platform project" error makes me think something's weird

**Comment by jonas0b1011001** [comment]
You are not the only one having issues: [https://issuetracker.google.com/issues/271559064](https://issuetracker.google.com/issues/271559064)

I guess you have to wait for google to fix it or try on a different account if you have one.
